dbt-bigquery/tests/unit/test_bigquery_adapter.py

```diff
@@ -419,7 +427,7 @@ def test_cancel_open_connections_single(self):
         self.assertEqual(len(list(adapter.cancel_open_connections())), 1)
 
     @patch("dbt.adapters.bigquery.clients.ClientOptions")
-    @patch("dbt.adapters.bigquery.credentials.default")
+    @patch("dbt.adapters.bigquery.credentials._create_bigquery_defaults")
```

Comment:
When there was only one test touching this is was covering up the fact that it is cached. and so the first mock "locks in" its mock instances.